src/Functions/searchAnyAll.cpp

```diff
@@ -10,7 +10,7 @@
 #include <Interpreters/ITokenExtractor.h>
 #include <Common/FunctionDocumentation.h>
 
-#include <algorithm>
+#include <absl/container/flat_hash_map.h>
```

Comment:
C++-23 added `std::flat_map` some reason to no use it?

Reply:
I guess we haven't started to use C++23 features actively in the codebase. Therefore, I'd stick with something used frequently in the codebase.